nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -375,6 +375,7 @@ def inference_step(self, dataloader_iter, batch_idx, mode, dataloader_idx=0):
             "add_BOS": False,
             "all_probs": False,
             "compute_logprob": False,
+            "end_strings": ["<|endoftext|>", "<extra_id_1>"],
```

Comment:
There is some inconsistency between here and above about "default" end strings. Let us use only <|endoftext|> everywhere.

Reply:
Removed.